### DISCUSSION TITLE
($100 bounty) Integrating JupyterLite into Vue project for students

### DISCUSSION THREAD

**Discussion by cameronweibel** [jupyterlite/jupyterlite#908] [General]
We have an app built on Vue we are using with high school students to learn various programming language. We can serve JupyterLite as described in docs, but we need to be able to:

1. Put the index.html into a simple Vue component so we can load exercises from our database
2. Use seaborn as a built-in package for the students.

If any of the core contributors can help us set up the above over Discord (or do it in a code sandbox and send it over), we will pay $100. I don't believe it will take much time. Thanks!

**Comment by JackYinpei** [comment]
100 dollars is too little to achieve this functionality. I believe it would take at least 1000 dollars to accomplish it.
